libs/zopengl/src/wrapper.zig

```diff
@@ -0,0 +1,2637 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+pub const bindings = @import("bindings.zig");
+
+pub const Framebuffer = packed struct { _: Uint = 0 };
```

Comment:
Maybe it would be better to use `extern struct` or `packed struct(u32)`?

Reply:
I think we should name the member `name` because this is how OpenGL refers to resource ids.
```zig
pub const Buffer = extern struct { name: Uint = 0 };
```

Reply:
Looks good. I was unsure about `packed` vs `extern` here.

In this case, do you think the `@bitCast`s should be changed to `handle.name` or it doesn't matter?

Reply:
When using `extern struct` it is guranteed that there won't be any padding (this C rule). We don't need to check the size in this case.

I think that `handle.name` would be cleaner than `@bitCast()` but I don't have strong opinion.